### PR TITLE
update changelog

### PR DESCRIPTION
**Pull request by BoWang2**

### CODE REVIEW

**Comment by BoWang2** [comment]
@dextertd please review this pull request. Thank you

**Comment by dextertd** [comment]
@BoWang2 Is it better for users to always use `/v2/private/position/switch-isolated` rather than `/v2/private/position/leverage/save`? They seem to fulfil the same purpose and have the same parameters.

File: locales/en.yml [hunk 1]
```diff
@@ -725,7 +725,7 @@ en:
   ### Set Leverage
   # see setleverage
   setleverage: Set Leverage
-  account_aside_setleverage: Set <code>leverage</code> to <code>0</code> if you want to trade in <a href="https://help.bybit.com/hc/en-us/articles/360037683633-What-is-Cross-Margin">Cross Margin mode</a>. Note that your position mode will be changed to Isolated Margin mode if you change your leverage from 0 to any other value.
+  account_aside_setleverage: This endpoint only concerns isolated margin. If you want to use cross margin, please use the <a href="Cross/Isolated Margin Switch"></a> endpoint.
```
Comment (dextertd) [inline locales/en.yml:728]: @BoWang2 Please translate the equivalent `account_aside_setleverage` in `zh-cn.yml`

**Review by dextertd — CHANGES_REQUESTED** [review]

File: locales/en.yml (hunk 1 above)
Comment (dextertd) [reply locales/en.yml:728]: Once that is done we can merge

Comment (BoWang2) [reply locales/en.yml:728]: have Done

**Comment by EricZ11** [comment]
> @BoWang2用户总是使用`/v2/private/position/switch-isolated`而不是更好`/v2/private/position/leverage/save`吗？它们似乎实现相同的目的并具有相同的参数。

/v2/private/position/leverage/save will update please review.

**Review by dextertd — APPROVED** [review]

**Pull request merged**